spark/src/main/scala/geotrellis/spark/tiling/MapKeyTransform.scala

```diff
@@ -95,12 +104,16 @@ class MapKeyTransform(val extent: Extent, val layoutCols: Int, val layoutRows: I
     (tcol.floor.toInt, trow.floor.toInt)
   }
 
-  def apply[K: SpatialComponent](key: K): Extent = {
-    apply(key.getComponent[SpatialKey])
-  }
+  @deprecated("Use this instead: MapKeyTransform.keyToExtent(key.getComponent[SpatialKey])", "1.2")
+  def apply[K: SpatialComponent](key: K): Extent = apply(key.getComponent[SpatialKey])
+
+  /** Get the [[Extent]] corresponding to a [[SpatialKey]] in some zoom level. */
+  def tileExtent(key: SpatialKey): Extent = apply(key)
```

Comment:
As mentioned above, I think we should keep away from naming things with `tile`.

Suggestion: `keyExtent`

Reply:
It was previously `toExtent`. Which do you think is better?

Reply:
After a talk with @echeipesh , the current vote is `SpatialKey.extent` (maximum minimalism, the word "key" doesn't need to be repeated since that's implied by the parent class), and `keyToExtent` for the `MapKeyTransform` methods so that they match the naming of their sibling methods. Thoughts?

Reply:
:+1: